File: tests/fixtures/patterns-positive/observer/mutB.ts
// POSITIVO (mutación B: sin colección de suscriptores; llamadas directas).
class Subject {
  private a: ObserverA;
  private b: ObserverB;
  constructor(a: ObserverA, b: ObserverB) {
    this.a = a;
    this.b = b;
  }
  notifyAll(event: string): void {
    this.a.update(event);
    this.b.update(event);
  }
}

File: tests/fixtures/patterns-positive/observer/mutB.py
# POSITIVO (mutación B: sin colección de suscriptores; llamadas directas).
class Subject:
    def __init__(self, a, b):
        self.a = a
        self.b = b

    def notify_all(self, event):
        self.a.update(event)
        self.b.update(event)
